tests/test_docker_ops.py

```
@@ -18,35 +18,32 @@ def _completed(returncode: int = 0, stdout: str = "", stderr: str = "") -> subpr
+    return DockerOperations("bedrock", Path("/srv/project"), executor=executor), executor
 
 
 # ---------------------------------------------------------------------------
 # Status tests
 # ---------------------------------------------------------------------------
 
-@patch("minecraft_manager.docker_ops.subprocess.run")
-def test_status_running(mock_run, ops: DockerOperations) -> None:
-    mock_run.return_value = _completed(stdout="running\n")
+def test_status_running() -> None:
+    ops, executor = _ops(_completed(stdout="running\n"))
```

Ruff (0.16.1)

[warning] Unpacked variable `executor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)